submissions/Vlada-khom/html-css-popup/index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <title>Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav>
+        <ul>
+          <li>
+            <a href="#" class="navLink">Mail</a>
+          </li>
+          <li>
+            <a href="#" class="navLink">Images</a>
+          </li>
+          <li class="burger">
+            <input type="checkbox" id="burgerCheckbox" class="input" />
+            <label for="burgerCheckbox" class="burgerLabel navLink">
+              <img
+                class="imgLink burgerImg"
+                src="./img/popup-button.png"
+              />
```

Comment:
![pr](https://user-images.githubusercontent.com/79094372/184347125-1adf9a7a-948e-417a-8b8b-fb795173d8dc.png)
we have already discussed this question, haven't we or not? In this case, I don't have alt, because it's icon for button.
![image](https://user-images.githubusercontent.com/79094372/184347983-3abbc815-f931-4cd5-ba7c-e13e6a21e479.png)

Reply:
sorry, I forgot to replace index.html, I added changes only to my repo but not in this PR.